src/query/service/src/servers/http/v1/query/http_query.rs

```diff
@@ -235,7 +235,8 @@ impl HttpQuery {
             }
             session
         } else {
-            ctx.get_session(SessionType::HTTPQuery)
+            ctx.upgrade_session(SessionType::HTTPQuery)
```

Comment:
Better to add a http api unit test to check the max sessions.

Reply:
working on it

Reply:
done